package.json

```diff
@@ -38,6 +38,7 @@
     "openapi-types": "^12.1.0",
     "tsup": "^6.5.0",
     "typescript": "^4.9.4",
+    "vite": "^4.0.4",
```

Comment:
this can be omitted, it declared in `example/package.json`

Reply:
if exclude `vite` and run `pnpm install && pnpm test`, it will prompt the following error message
![image](https://user-images.githubusercontent.com/25837029/231941935-8f367a8a-024e-4548-9898-495144622f35.png)
because example and the main package are not in a workspace and I don't think putting them in a workspace is good practice

Reply:
wait, vitest already depends on vite and you don't have to manually install vite to use vitest iirc.

Reply:
I tried `pnpm install && pnpm test` with fresh clone, it works fine for me, what's your pnpm version?

Reply:
It's `7.29` before.After upgrading to `7.32`, problem solved.😄